Sources/Purchasing/Configuration.swift

```diff
@@ -238,6 +238,7 @@ extension Configuration {
 
         case productRequest = 3
         case purchase = 7
+        case introEligibility = 2
```

Comment:
I've chosen 2 seconds but let me know if you think that's too low?

Reply:
> I'd make it a bit higher since we've had cases where it takes > 30 secs, but it's entirely arbitrary so I'm good with any value here. 

I added intro eligibility checking to my app and suddenly my prices were taking a lot longer to show up. I immediately knew why, but some users might think there's something wrong. My thought was that anything beyond 2 seconds might be useful to let the developer know where the delay in the SDK was.

> We should make sure that the logs do state that it's taking longer than X secs, though, so it's clear to a dev what "longer than expected" means

That's done automatically by `TimingUtil` :) this is an example log:

> WARN: 🍎⚠️ StoreKit 2 intro eligibility took longer than expected to determine. (2.03 seconds)

Reply:
oh nice, I forgot about that